code/modules/surgery/tooth.dm

```diff
@@ -0,0 +1,221 @@
+	"<span class='notice'>You mend [target]'s dental sockets with \the [tool].</span>")
+	target.op_stage.tooth = 1
+
+/datum/surgery_step/tooth/mend_roots/fail_step(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
+	var/datum/organ/external/affected = target.get_organ(target_zone)
+	user.visible_message("<span class='warning'>[user]'s hand slips, ripping a bare nerve out of [target]'s jaw with \the [tool]!</span>", \
+	"<span class='warning'>Your hand slips, ripping a bare nerve out of [user]'s jaw with \the [tool]!</span>")
+	target.apply_damage(5, BRUTE, affected)
+
+
+
+////////ADD NEW TEETH//////
+/datum/surgery_step/tooth/new_teeth
+	allowed_tools = list(
+		/obj/item/stack/teeth = 100,
+		)
+
+	min_duration = 60
+	max_duration = 80
+
+/datum/surgery_step/tooth/new_teeth/can_use(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
+
+	var/teeth_missing
+
+	var/datum/butchering_product/teeth/J = locate(/datum/butchering_product/teeth) in target.butchering_drops
+	if(!istype(J))
+		return 0
+	if(J.amount < J.initial_amount)
+		teeth_missing = 1
+	else
+		to_chat(user, "<span class='warning'>\The [target] already has a full mouth of teeth.</span>")
+		return 0
+
+	return ..() && target.op_stage.tooth == 1 && teeth_missing
+
+/datum/surgery_step/tooth/new_teeth/begin_step(mob/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
+	user.visible_message("[user] starts carefully planting \a [tool] in [target]'s jaw.", \
+	"You start carefully planting \a [tool] in [target]'s jaw.")
+	..()
+
+/datum/surgery_step/tooth/new_teeth/end_step(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
+	user.visible_message("<span class='notice'>[user] has finished planting \a [tool] in [target]'s jaw.</span>",	\
+	"<span class='notice'>You have finished planting \a [tool] in [target]'s jaw.</span>")
+	var/obj/item/stack/teeth/T = tool
+	var/datum/butchering_product/teeth/J = locate(/datum/butchering_product/teeth) in target.butchering_drops
+	J.amount += 1
+	T.amount -= 1
+	if(T.amount <= 0)
+		user.before_take_item(T)
+
+
+/datum/surgery_step/tooth/new_teeth/fail_step(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
+	var/datum/organ/external/affected = target.get_organ(target_zone)
+	user.visible_message("<span class='warning'>[user]'s hand slips, cutting [target]'s gums with \a [tool]!</span>", \
+	"<span class='warning'>Your hand slips, cutting [target]'s gums with \a [tool]!</span>")
+	target.apply_damage(5, BRUTE, affected)
+
+
+//////////////////////////////////////////////////////////////////
+//						TOOTH EXTRACTION SURGERY							//
+//////////////////////////////////////////////////////////////////
+
+/datum/surgery_step/extract
+	priority = 5
+	can_infect = 0
+	can_use(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
```

Comment:
@PJB3005 
This is the way all of the surgeries are coded. I literally lifted that line directly out of the face surgery code. If you want to go and fix every single surgery, by all means go ahead. I'm not going to.